feat(content): Short Rulei Mission - Umbral Reach (#8597)
endless-sky/endless-sky: data/rulei/rulei.txt

	on offer
		fail "Ira Contact"
	on enter "Kanguwa"

> 		fail

Reviewer: Why is this removed?

Author: Good question.

			`	It's best you don't remain here for too long.`
				decline

mission "Umbral's Reach"
	invisible
	landing
	to complete
		never
	to offer
		has "First Contact: Rulei: offered"
		has "visited system: L-118"
	on enter "Kanguwa"

> 		log "Factions" "Rulei" `The ancient Rulei ship stated to be careful of "Umbral's Reach".`

Reviewer: I don't think this is worth logging. See the Archon nanobots logbook entry: it doesn't mention to avoid Sayaiban, but it instead logs the insight behind the Archon's actions.

Author: Good point.

		conversation

> 			`Out of nowhere, the mysterious whispers of the Rulei begin to seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direcetion of the ancient vessel, eventually coalescing into one spoken phrase.`

Reviewer: ```suggestion
			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
```

			`	"Beware.`
			`	"Umbral's Reach."`

> 			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`

Reviewer: ```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once -- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```

			`	It's best you don't remain here for too long.`
				decline

mission "Umbral's Reach"
	invisible
	landing
	to complete
		never
	to offer
		has "First Contact: Rulei: offered"

> 		has "visited system: L-118"

Reviewer: Why does this system need to be visited?

Author: This system is the first system you pass through in the dark region area (and is the only way you can get through to it). The point of this mission is the vyu-Ir is able to tell you visited the area, and has a comment on it. Realistically, the player would have gone beyond L-118, but L-118 is still one of the Reach's systems.

	on enter "Kanguwa"
		conversation
			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
			`	"Beware..."`

> 			`	"...Umbral's Reach."`

Reviewer: Does this have something to do with the system Umbral (connected to Tarazed), or are they unrelated?

Author: No. "Umbral Reach" is the name for the dark region that was decided after a bit of discussion with others. Void, Abyss, etc. will be used in Mata content as far as I know, and anything else we came up didn't really fit it. Perhaps I reword this back to "Umbral Reach" so it doesn't possibly imply that "Umbral is reaching", as in the planet?

Reviewer: That sounds good.
```suggestion
			`	"...Umbral Reach."`
```

> 			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`

Reviewer: The last sentence doesn't add anything.
```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence.`
```

> 			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once -- sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`

Reviewer: ```suggestion
			`	You feel tremendous weight behind the expression, as if it shook your mind itself. It's as though every emotion has flooded into your head at once - sadness, anger, joy, disgust, even some you can't seem to understand. Before you can respond to them, the voices quickly dissipate, and you're left alone in an uncomfortable silence. You aren't sure what "Umbral's Reach" is, but it seems the Rulei don't want you to be involved with it.`
```
